indexing-service/src/main/java/org/apache/druid/indexing/common/IngestionStatsAndErrorsTaskReportData.java

```diff
@@ -47,13 +47,37 @@ public class IngestionStatsAndErrorsTaskReportData
   @JsonProperty
   private long segmentAvailabilityWaitTimeMs;
 
+  @JsonProperty
+  private Map<Object, Long> partitionStats;
+
+  public IngestionStatsAndErrorsTaskReportData(
```

Comment:
I believe the convention in Druid is to avoid overriding constructors if possible.